discover/portal/articles/130-forms/03-form-rules/00-intro.markdown

```diff
@@ -0,0 +1,111 @@
+# Form Rules
+
+What's the difference between a chicken and a dog? Among other things, you can't
+train a chicken, while dogs are much more trainable. Don't believe me? Try
+teaching your chicken to sit on command or herd sheep. Better yet, get a team of
```

Comment:
Herd is a verb in this case. To herd sheep is to bring them together, and some dogs are trained to do this. Let me know if you think this will be confusing and I should change it. https://www.youtube.com/watch?v=MCEcy-zjHeo
:)